src/FirstSteps/RPG/Adventures/Mine.cs

```diff
@@ -14,40 +14,65 @@ public void Enter(Hero hero)
                 Display.ErrorText("You can't enter! Only Dwarf can enter to the Mine!");
                 return;
             }
-
             Display.DefaultText("You entered to the Mine! ");
 
             enter += 1;
-
-            var coins = enter % 2 == 0 ? 3 : 1;
+            var coins = 1;
+            if (enter % 2 == 0)
+            {
+                coins = 3;
+                DisplayCoins(coins);
+                hero.AddCoins(coins);
+            }
+            else if (enter % 5 == 0)
+            {
+                coins += 2;
+                DisplayCoins(coins);
+                hero.AddCoins(coins);
+            }
+            else if (enter == 13)
```

Comment:
This condition is some kind of exception so you can check it at the beginning and just end the method with `return;`.